scripts/kfctl.sh

```diff
@@ -178,6 +181,10 @@ parseArgs() {
         shift
         EMAIL=$1
         ;;
+      --gkeApiVersion)
```

Comment:
Why do we need to expose gkeApiVersion as a command line argument? Can't users just modify the YAML config to set it? 

In the E2E tests we can just use yq or similar to script modifying the config as needed.

In general, I'd like to avoid plumbing more and more of the options in the YAML config file as command line arguments. I don't think that really scales.